website/pages/component/[id]/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { GetStaticPaths, GetStaticProps } from 'next';
+import fs from 'fs';
+import path from 'path';
+import Layout from '../../../components/layout';
```

Comment:
Do we intend this component to be re-used elsewhere?

If no, then really we should just inline it in here.

Reply:
👍